config/krakend/partials/tenant_headers.tmpl

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I would rename the file to default_tenant_headers

Reply:
name changed